module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -737,6 +739,41 @@ function ($a, $b) use ($sortField) {
         return $holdings;
     }
 
+    /**
+     * Get all bib records bound-with this item, including
+     * the directly-linked bib record.
+     * 
+     * @param object $item The item record
+     * 
+     * @return array An array of key metadtaa for each bib record
+     */
+    protected function getBoundWithRecords($item)
+    {
+        $boundWithRecords = [];
+        // Get the bound-with holdings for the item.
+        foreach (
+            $this->getPagedResults(
+                'boundWithParts',
+                '/inventory-storage/bound-with-parts',
+                ['query' => '(itemId=="' . $item->id . '")']
+            ) as $boundWithPart
+        ) {
+            $boundWithHoldingId = $boundWithPart->holdingsRecordId;
+            $response = $this->makeRequest(
+                'GET',
+                '/holdings-storage/holdings/' . $boundWithHoldingId
+            );
+            $holding = json_decode($response->getBody());
+            // Get the bound-with holding's instance record.
+            $instance = $this->getInstanceById($holding->instanceId);
+            $boundWithRecords[] = [
+                'title' => $instance->title,
+                'bibId' => $this->getBibId($instance),
+            ];
```

Comment:
If you think it's important, I could try to figure out how to contain each $instance in a RecordDriver, in case a template wants something else out of the instance.  Or I suppose just put the instance itself in the array.  Neither seemed necessary but I'm not the best judge.

Reply:
We don't typically load record drivers inside ILS drivers, so I think it's fine to just export the IDs. If we need record drivers, we should probably fetch them "on the outside" anyway.